nautobot/core/templates/inc/media.html

```diff
@@ -20,8 +20,11 @@
           href="{% static 'select2-bootstrap-0.1.0-beta.10/select2-bootstrap.min.css' %}"
           onerror="window.location='{% url 'media_failure' %}?filename=select2-bootstrap-0.1.0-beta.10/select2-bootstrap.min.css'">
     <link rel="stylesheet"
-          href="{% static 'flatpickr-4.6.9/themes/light.min.css' %}"
-          onerror="window.location='{% url 'media_failure' %}?filename=flatpickr-4.6.9/themes/light.min.css'">
```

Comment:
Was this removal intentional?

Reply:
Absolutely not, thanks for spotting that one!